src/cryptography/x509.py

```diff
@@ -1158,3 +1225,18 @@ def signature_hash_algorithm(self):
         Returns a HashAlgorithm corresponding to the type of the digest signed
         in the certificate.
         """
+
+
+@six.add_metaclass(abc.ABCMeta)
+class RevokedCertificate(object):
```

Comment:
I'm okay with incrementally adding things in future PRs so if you want to leave extensions out for the moment that's okay. We do want to keep master releasable at all times so if there's a case where it would do the "wrong" thing without supporting something we can implement a temporary exception for that.